src/GoogleSignIn.ts

```diff
@@ -255,19 +261,29 @@ class GoogleSignIn implements GoogleOneTapAuthPlugin {
     invisibleGoogleButtonDiv.style.width = `${buttonElemWidth}px`;
     invisibleGoogleButtonDiv.style.boxSizing = 'border-box';
     invisibleGoogleButtonDiv.style.opacity = '0.0001'; // change it for debugging
+
     buttonElem!.appendChild(invisibleGoogleButtonDiv);
 
+    // Since we can't set the height on the Google sign-in button, we'll just scale everything up vertically.
+    const googleButtonHeight = 40;
+    const scaleY = buttonElemHeight / googleButtonHeight;
+
+    // Scale the invisible button so that the clickable area will be tall enough, such as if our button has vertical
+    // padding. Otherwise, the bottom of our button may not respond to clicks.
+    invisibleGoogleButtonDiv.style.transformOrigin = 'top left';
```

Comment:
My preference would be to set all properties of invisibleGoogleButtonDiv before it is appended to buttonElem. So move the 2 lines up. Then the calculation of scaleY can be moved above the line "const invisibleGoogleButtonDiv = document.createElement('div');"